cmd/zdx/section/command.go

```diff
@@ -0,0 +1,106 @@
+package section
+
+import (
+	"errors"
+	"flag"
+	"os"
+
+	"github.com/brimsec/zq/cmd/zdx/root"
+	"github.com/brimsec/zq/emitter"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zdx"
+	"github.com/brimsec/zq/zio"
+	"github.com/brimsec/zq/zng/resolver"
+	"github.com/mccanne/charm"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var Section = &charm.Spec{
+	Name:  "section",
+	Usage: "section [flags] path",
+	Short: "extract a section of a zdx file",
+	Long: `
+The section command extracts a section from a microindex file and
+writes it to the output.  The -trailer option writes
+the microindex trailer to the output in addition to the section if the section
+number was specified.
+
+See the zdx command help for a description of a microindex.`,
```

Comment:
`zdx help` refers to boom sst files and doesn't say "microindex"... time to update that one I think 🙂